vignettes/config.Rmd

```diff
@@ -84,3 +90,5 @@ hugodown does not work with every possible hugo site. There is some config that
 hugodown also has it's own configuration file, `_hugodown.yml`. Currently this has one option:
 
 * `hugo_version`: this defines the version of hugo needed by the current site.
+
+[styles]: https://xyproto.github.io/splash/docs/longer/all.html
```

Comment:
Oops!

Reply:
I am actually curious why you prefer putting URLs at the bottom?

Reply:
I don't usually, but sometimes it makes the flow of a paragraph ugly.